tests/integration/src/test/resources/org/mule/test/integration/exceptions/catch-exception-strategy-transaction-flow.xml

```diff
@@ -41,4 +41,15 @@
         <catch-exception-strategy/>
     </flow>
 
+    <flow name="transactionCommitFails">
+        <vm:inbound-endpoint path="in3">
+            <vm:transaction action="ALWAYS_BEGIN"/>
+        </vm:inbound-endpoint>
+        <test:component />
```

Comment:
Yes, I need it because I use it to change the current transaction in the flow and fail in some scenarios.